airflow/models/dag.py

```diff
@@ -661,6 +661,15 @@ def get_last_dagrun(self, session=None, include_externally_triggered=False):
             self.dag_id, session=session, include_externally_triggered=include_externally_triggered
         )
 
+    @provide_session
+    def has_no_dag_runs(self, session=None, include_externally_triggered=True) -> bool:
```

Comment:
I have mixed feelings about methods that have negations in their names. What do you think about "has_dag_runs"?

Reply:
I totally agree—not sure what I was thinking 😄